client/dashboard/components/cart-modal.js

```diff
@@ -37,13 +37,12 @@ class CartModal extends Component {
 		const { productIds, onClickPurchaseNow } = this.props;
 		this.setState( { purchaseNowButtonBusy: true } );
 		if ( ! productIds.length ) {
-
 			return;
 		}
 
 		recordEvent( 'tasklist_modal_proceed_checkout', {
 			product_ids: productIds,
-			purchase_install: false,
+			purchase_install: true,
```

Comment:
Should this just be removed to avoid confusion when looking at Tracks later on?

Reply:
Not quite sure I follow.  This event prop helps track if they closed the modal without purchasing or opted to purchase now.